Wrapping/Python/Packaging/setup.py.in

```diff
@@ -90,5 +108,6 @@ setup(
         ],
     license='Apache',
     keywords = 'ITK InsightToolkit segmentation registration image',
```

Comment:
Should the keywords also include SimpleITK? Some additional potential keywords  'analysis biomedical medical remote sensing'